# Patch generated by Pyment v0.3.3

--- a/..//venv/lib/python3.8/site-packages/pandas/tests/frame/methods/test_to_records.py
+++ b/..//venv/lib/python3.8/site-packages/pandas/tests/frame/methods/test_to_records.py
@@ -15,7 +15,9 @@
 
 
 class TestDataFrameToRecords:
+    """ """
     def test_to_records_dt64(self):
+        """ """
         df = DataFrame(
             [["one", "two", "three"], ["four", "five", "six"]],
             index=date_range("2012-01-01", "2012-01-02"),
@@ -26,6 +28,7 @@
         assert expected == result
 
     def test_to_records_dt64tz_column(self):
+        """ """
         # GH#32535 dont less tz in to_records
         df = DataFrame({"A": date_range("2012-01-01", "2012-01-02", tz="US/Eastern")})
 
@@ -37,6 +40,7 @@
         assert val == df.loc[0, "A"]
 
     def test_to_records_with_multindex(self):
+        """ """
         # GH#3189
         index = [
             ["bar", "bar", "baz", "baz", "foo", "foo", "qux", "qux"],
@@ -49,6 +53,7 @@
         assert "one" not in r
 
     def test_to_records_with_Mapping_type(self):
+        """ """
         import email
         from email.parser import Parser
 
@@ -66,10 +71,12 @@
         all(x in frame for x in ["Type", "Subject", "From"])
 
     def test_to_records_floats(self):
+        """ """
         df = DataFrame(np.random.rand(10, 10))
         df.to_records()
 
     def test_to_records_index_name(self):
+        """ """
         df = DataFrame(np.random.randn(3, 3))
         df.index.name = "X"
         rs = df.to_records()
@@ -85,6 +92,7 @@
         assert "level_0" in rs.dtype.fields
 
     def test_to_records_with_unicode_index(self):
+        """ """
         # GH#13172
         # unicode_literals conflict with to_records
         result = DataFrame([{"a": "x", "b": "y"}]).set_index("a").to_records()
@@ -92,6 +100,7 @@
         tm.assert_almost_equal(result, expected)
 
     def test_to_records_with_unicode_column_names(self):
+        """ """
         # xref issue: https://github.com/numpy/numpy/issues/2407
         # Issue GH#11879. to_records used to raise an exception when used
         # with column names containing non-ascii characters in Python 2
@@ -106,6 +115,7 @@
         tm.assert_almost_equal(result, expected)
 
     def test_to_records_with_categorical(self):
+        """ """
         # GH#8626
 
         # dict creation
@@ -258,6 +268,19 @@
         ],
     )
     def test_to_records_dtype(self, kwargs, expected):
+        """
+
+        Parameters
+        ----------
+        kwargs :
+            
+        expected :
+            
+
+        Returns
+        -------
+
+        """
         # see GH#18146
         df = DataFrame({"A": [1, 2], "B": [0.2, 1.5], "C": ["a", "bc"]})
 
@@ -331,13 +354,30 @@
         ],
     )
     def test_to_records_dtype_mi(self, df, kwargs, expected):
+        """
+
+        Parameters
+        ----------
+        df :
+            
+        kwargs :
+            
+        expected :
+            
+
+        Returns
+        -------
+
+        """
         # see GH#18146
         result = df.to_records(**kwargs)
         tm.assert_almost_equal(result, expected)
 
     def test_to_records_dict_like(self):
+        """ """
         # see GH#18146
         class DictLike:
+            """ """
             def __init__(self, **kwargs):
                 self.d = kwargs.copy()
 
@@ -348,6 +388,7 @@
                 return key in self.d
 
             def keys(self):
+                """ """
                 return self.d.keys()
 
         df = DataFrame({"A": [1, 2], "B": [0.2, 1.5], "C": ["a", "bc"]})
@@ -366,6 +407,17 @@
 
     @pytest.mark.parametrize("tz", ["UTC", "GMT", "US/Eastern"])
     def test_to_records_datetimeindex_with_tz(self, tz):
+        """
+
+        Parameters
+        ----------
+        tz :
+            
+
+        Returns
+        -------
+
+        """
         # GH#13937
         dr = date_range("2016-01-01", periods=10, freq="S", tz=tz)
 
